infra/concourse/build/developer-tools/build/scripts/task_wrapper_scripts/check_bash

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+source /usr/local/bin/task_helper_functions.sh && check_bash
```

Comment:
We need to look into this, I get what looks to be an infinite recursion bug:

```bash
docker run --rm -e SERVICE_ACCOUNT_JSON -it -v `pwd`:/workspace cft/developer-tools:0.0.1
bash-5.0# source /usr/local/bin/task_helper_functions.sh
bash-5.0# cd /workspace/
bash-5.0# check_bash
/bin/bash: warning: shell level (1000) too high, resetting to 1
/bin/bash: warning: shell level (1000) too high, resetting to 1
/bin/bash: warning: shell level (1000) too high, resetting to 1
/bin/bash: warning: shell level (1000) too high, resetting to 1
^C
bash-5.0#
```

Reply:
Fixed it in 39c8b0ee8b9a420638c01dc34d103932abdf05fc Remove bugged check_bash, use check_shell instead